go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/lbryio/lbry.go/v2
 
+replace github.com/btcsuite/btcd => github.com/lbryio/lbrycrd.go v0.0.0-20200203050410-e1076f12bf19
```

Comment:
This is how you use a forked version of something without messing with all the imports everywhere.